src/wp-includes/rest-api/class-wp-rest-server.php

```diff
@@ -1341,7 +1341,7 @@ protected function respond_to_request( $request, $route, $handler, $response ) {
 	protected function get_json_last_error() {
 		$last_error_code = json_last_error();
 
-		if ( JSON_ERROR_NONE === $last_error_code || empty( $last_error_code ) ) {
+		if ( empty( $last_error_code ) || JSON_ERROR_NONE === $last_error_code ) {
```

Comment:
As noted in [the trac ticket](https://core.trac.wordpress.org/ticket/63249#comment:6) by @khushdoms, this check has been simplified in [r60151](https://core.trac.wordpress.org/changeset/60151).

It's causing this file's patch to fail to apply -- I'd suggest removing this particular change, to allow the PR / patch to apply cleanly